e2e/specs/typography.spec.js

```diff
@@ -26,13 +26,45 @@ describe("the typography in the main block", () => {
       .invoke("css", "opacity", "1");
   });
 
-  it("matches the snapshot", () => {
-    cy.get(".main").matchThemedSnapshots("main-block");
+  it("matches the snapshot for main vs sidebar", () => {
+    cy.get("body").matchThemedSnapshots("main-vs-sidebar");
```

Comment:
Something weird is going on with this snapshot as it seems to get cut off well before when you'd expect it to be

Reply:
Oh this is OK. This snapshot is just trying to capture the top of the screen anyway, to make sure all the top alignment is correct. I'll rephrase the test description to clarify.